test/cucumber_test.go

```diff
@@ -65,7 +65,9 @@ func InitializeScenario(ctx *godog.ScenarioContext) {
 				fmt.Println("Error while killing app process:", err.Error())
 			}
 			if err := app.Wait(); err != nil {
-				fmt.Println("Error while waiting for app to exit:", err.Error())
+				if err.Error() != "signal: killed" {
```

Comment:
This is just to suppress an error being printed when the app has aactually already been successfully killed